ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/visitor/core/interfaces/visitor/IMethodAnalyzerVisitor.java

```diff
@@ -0,0 +1,12 @@
+package edu.usb.argos.ASTProcessor.visitor.core.interfaces.visitor;
+
+import edu.usb.argos.ASTProcessor.visitor.core.entities.method.*;
```

Comment:
It's not good to use * when importing due to we access to the all library so please import just what you need.

Reply:
Done